src/main/java/de/tum/in/www1/artemis/web/rest/UserResource.java

```diff
@@ -196,6 +196,30 @@ public ResponseEntity<UserDTO> updateUser(@Valid @RequestBody ManagedUserVM mana
         return ResponseEntity.ok().headers(HeaderUtil.createAlert(applicationName, "userManagement.updated", managedUserVM.getLogin())).body(new UserDTO(updatedUser));
     }
 
+    /**
+     * PUT ldap : Updates an existing User based on the info available in the LDAP server.
+     *
+     * @param login of the user to update
+     * @return the ResponseEntity with status 200 (OK) and with body the updated user
+     */
+    @PutMapping("users/{login:" + Constants.LOGIN_REGEX + "}/ldap-sync")
+    @PreAuthorize("hasRole('ADMIN')")
```

Comment:
please add the profile LDAP, so that this method is only available with an active LDAP profile

Reply:
Profile added.